driver/controller.go

```diff
@@ -253,6 +261,8 @@ func (d *Driver) ControllerPublishVolume(ctx context.Context, req *csi.Controlle
 		return nil, err
 	}
 
+	d.tagVolume(ctx, vol)
```

Comment:
We're not checking for the error here.

Reply:
Addressed.  In an earlier iteration I was contemplating making these updates asynchronously so as to not scuttle a volume attach.  I changed that approach, but forgot to handle the error value up here.  

I wasn't sure what the best code would be, so I went with `internal`. https://github.com/digitalocean/csi-digitalocean/pull/130/commits/eea3ef3c35041711cf61ed12ee2da965322e7ae8#diff-aa5513ce5e9cac724e0d7994151fed94R267